src/adapters/search_suggestions_adapter.js

```diff
@@ -1,26 +1,49 @@
-import webChannel from '../lib/web_channel';
+import State from 'ampersand-state';
 import _c from 'lodash/collection';
 import SearchSuggestions from '../collections/search_suggestions';
+import webChannel from '../lib/web_channel';
+
+const SearchSuggestionsAdapter = State.extend({
+  props: {
+    engine: 'string',
+    searchTerm: 'string'
+  },
+
+  initialize () {
+    this.maxLocalSuggestions = 2;
+    this.maxCombinedSuggestions = 5;
```

Comment:
I know nothing, but to my ignorant eyes, it seems a bit scary to hardcode values into files.
Should these be in some `const` values somewhere and imported, or in some convict config?